Documentation/ApiOverview/Fal/Administration/Maintenance.rst

```diff
@@ -60,3 +60,75 @@ Here you can choose to delete all files in :file:`fileadmin/_processed_/`
 
 This cleanup is also good if processed files have accumulated for a
 long time. Many of them may then be obsolete.
+
+.. attention::
+
+    If you delete processed files, you should flush the (page) cache immediately
+    afterwards. If pages are cached and the page uses processed images, these
+    will not be regenerated on the fly when a page is loaded. Ideally, make sure
+    the removal of the processed files and flushing of page cache is one atomic
+    operation which is performed as quickly as possible.
+
+Also, deleting processed files while editors are active is not ideal.
+Preferably, lock the TYPO3 Backend before you remove the processed files.
+
+See also the next section in case you have many processed files and removal
+would take long (e.g. several minutes).
+
+Bulk removal of processed files
+-------------------------------
+
+This entire section is a "use at your own risk" hint.
+
+In general remove processed files as one atomic operation together with flushing
+the page cache afterwards.
+
+Removing processed files may take long if there are many. As a shortcut, you may
+remove all at once, but always be sure to consider the database table
```

Comment:
how?

Reply:
> how?
That is explained in the following text.

Reply:
I think it would be better to explain how to do it first and then add more why to do it

Reply:
After discussing via Slack, we decided to remove the bulk removal. I have stored the text locally. 

Reason: it is not really the recommended procedure and it bloats up the docs.

will push change shortly.